oss-internship-2020/curl/tests/callbacks.cc

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "callbacks.h"
+
+#include <cstdlib>
+#include <cstring>
+
+#include "sandboxed_api/vars.h"
+
+size_t WriteToMemoryTests(char* contents, size_t size, size_t num_bytes,
```

Comment:
This is the roughly same as WriteToMemoryCallback.
Can this be common for `tests` and `examples`?